web/docs/Backend Starters/Decoupled Drupal/creating-new-project.md

```diff
@@ -1,25 +1,162 @@
 ---
+id: "creating-new-drupal-project"
+title: "Creating a New Drupal Project"
+slug: "/Backend Starters/Decoupled Drupal/Creating a New Project"
 sidebar_position: 2
 ---
 
-# Creating a New Project
+## Choosing an Approach
 
-## Prerequisites
+### Use Build Tools if:
+
+- Testing is an important part of your workflow
+
+- You don’t want to worry about manually pushing changes to your GH repo.
+
+### Use Dashboard Upstream if:
+
+- The simplest possible setup
+
+- Pantheon repo is your source of truth
+
+- GitHub is your default VCS and CircleCI is your CI provider then this approach will automatically default to both these options. For other options use the build tools approach
```

Comment:
I don't think this statement is accurate. The empty upstream site doesn't really have any opinion on an external VCS used (and technically doesn't require one) and doesn't use CI.